tests/unit/cmorizers/obs/test_utilities.py

```diff
@@ -101,3 +107,205 @@ def test_fix_dtype_not_lazy(cube):
         if coord.has_bounds():
             assert coord.bounds_dtype == np.float64
     assert not is_lazy(cube)
+
+
+class VarInfoObj:
```

Comment:
Maybe you could use `unittest.mock.Mock()` to create a dummy VariableInfo object?